spec/spec_helper.rb

```diff
@@ -14,6 +14,7 @@
 require 'carrierwave'
 
 require 'pry'
+require "codeclimate-test-reporter"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.